pagopa/io-app
# feat(IT Wallet): [SIW-1025] Add credential issuance auth screen

## Short description
This PR introduces the credential issuance auth screen component with mocked data.

## List of changes proposed in this pull request
- Added reqiured assets
- Added required locale keys
- Removed `ItwBulletList` component and related data in `ItwMockUtils.ts`
- Removed extra spacing in `ItwMarkdown` paragraph component
- Added [ItwRequiredClaimsList.tsx](https://github.com/pagopa/io-app/compare/SIW-1025-credential-issuance-auth?expand=1#diff-98866b8ca3d8b56b2f54493834c67734136bc9e15d015bbec6437b1e35a0dd7b) component, which shows the list of required data to issue the credential
- Added [ItwIssuanceCredentialAuthScreen.tsx](https://github.com/pagopa/io-app/compare/SIW-1025-credential-issuance-auth?expand=1#diff-0dc769f6acff862613c9fe86808f897f58de29cd48332a0daa7582b848aebf7f)
- Added screen route, params and navigation component
- Added screen smoke test

## How to test
From the **Profile** section, navigate to **Playgrounds -> IT Wallet -> Credential auth (mDL)**, check that the component is rendered correctly

## Preview

https://github.com/pagopa/io-app/assets/6160324/693d0b41-8eb5-421b-8fbc-c80b4d5f25c3
